### PR TITLE
Jutzig/topic/csharp

### PR DESCRIPTION
**Pull request by cgrom**
fix when file for new translation has to be created

### CODE REVIEW

File: csharp/.classpath [hunk 1]
```diff
@@ -19,5 +19,6 @@
 		</attributes>
 	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry exported="true" kind="lib" path="C:/Program Files/Java/jdk1.8.0_202/lib/tools.jar"/>
```
Comment (jutzig) [inline csharp/.classpath]: This is incorrect, don't push that

Comment (jutzig) [inline csharp/.classpath]: This is incorrect, don't push that

File: csharp/src/main/java/org/jabylon/csharp/CSharpConverter.java [hunk 2]
```diff
@@ -51,6 +51,123 @@
 	private static final String TYPE = "type";
 	private static final String SYSTEM_STRING = "System.String";
 	private static final String INVARIANT = "@Invariant";			// denotes non translatable content
+	private static final String ROOT_NODE = "root";
+	private static final String EMPTY_RES_FILE = "<root>\r\n" +
```
Comment (jutzig) [inline csharp/src/main/java/org/jabylon/csharp/CSharpConverter.java]: needs to be extracted into a file

File: csharp/src/main/java/org/jabylon/csharp/CSharpConverter.java [hunk 3]
```diff
@@ -69,10 +186,11 @@ public CSharpConverter(URI resourceLocation, boolean prettyPrint) {
 
 	@Override
 	public PropertyFile load(InputStream in, String encoding) throws IOException {
+		LOG.debug("C#:load0");
```
Comment (jutzig) [inline csharp/src/main/java/org/jabylon/csharp/CSharpConverter.java]: these debug statements are pointless.
Also, log messages do not - and should not - contain location information. When the log is written, it contains class, method, and - if wanted - line numbers

File: csharp/src/test/java/org/jabylon/csharp/CSharpConverterTest.java [hunk 4]
```diff
@@ -1,55 +0,0 @@
...
-import org.eclipse.emf.common.util.EList;
-import org.eclipse.emf.common.util.URI;
-import org.jabylon.csharp.CSharpConverter;
-import org.jabylon.properties.Property;
-import org.jabylon.properties.PropertyFile;
-import org.junit.Before;
-import org.junit.Test;
-
-
-public class CSharpConverterTest {
```
Comment (jutzig) [inline csharp/src/test/java/org/jabylon/csharp/CSharpConverterTest.java]: why would you delete the tests?

File: csharp/src/test/java/org/jabylon/csharp/CSharpScannerTest.java [hunk 5]
```diff
@@ -1,113 +0,0 @@
...
-import java.util.Locale;
-import java.util.Map;
-
-import org.jabylon.properties.PropertiesFactory;
-import org.jabylon.properties.ScanConfiguration;
-import org.junit.Before;
-import org.junit.Test;
-
-
-public class CSharpScannerTest {
```
Comment (jutzig) [inline csharp/src/test/java/org/jabylon/csharp/CSharpScannerTest.java]: keep the test classes

**Review by jutzig — CHANGES_REQUESTED** [review]

File: csharp/src/main/java/org/jabylon/csharp/CSharpConverter.java (hunk 2 above)
Comment (cgrom) [reply csharp/src/main/java/org/jabylon/csharp/CSharpConverter.java]: extracted in file const.properties

**Pull request merged**